### PR TITLE
Dashboard links

### PR DESCRIPTION
**Pull request by teneightfive**
This change removes the need for client side code to show/hide links.

## Before

localhost_3001_ 1

## After

localhost_3001_

### CODE REVIEW

File: src/apps/dashboard/controllers.js [hunk 1]
```diff
@@ -6,12 +6,15 @@ function getHandler (req, res) {
...
     .then((data) => {
+      // TODO: Remove manual limit on interactions and contacts once API supports it
       res
         .title('Dashboard')
         .render('dashboard/views/dashboard', {
           totalDays: days,
-          interactions: data.interactions,
-          contacts: data.contacts,
+          interactionsCount: data.interactions.length,
+          interactions: data.interactions.slice(0, 5),
```
Comment (feedmypixel) [inline src/apps/dashboard/controllers.js]: are we always going to have data to `slice`?

File: src/apps/dashboard/controllers.js [hunk 2]
```diff
@@ -6,12 +6,15 @@ function getHandler (req, res) {
...
+          contactsCount: data.contacts.length,
+          contacts: data.contacts.slice(0, 5),
```
Comment (feedmypixel) [inline src/apps/dashboard/controllers.js]: same as above

File: src/apps/dashboard/controllers.js [hunk 3]
```diff
@@ -6,12 +6,15 @@ function getHandler (req, res) {
   dashboardService
     .getHomepageData(req.session.token, days)
```
Comment (feedmypixel) [inline src/apps/dashboard/controllers.js]: feels like we could do with a `limit=10` rather `days` on this

Comment (feedmypixel) [reply src/apps/dashboard/controllers.js]: @reupen FYI ☝️

Comment (teneightfive) [reply src/apps/dashboard/controllers.js]: I would agree as the response to the dashboard is currently very slow as it's returning ~200 items.

I would say for now we should limit to last 5 and we can revisit properly when we look at what the dashboard should actually contain.

Comment (feedmypixel) [reply src/apps/dashboard/controllers.js]: > I would say for now we should limit to last 5 and we can revisit properly when we look at what the dashboard should actually contain.

👍

**Pull request merged**